torch/csrc/jit/passes/quantization/helper.cpp

```diff
@@ -668,5 +668,17 @@ bool is_batchnorm3d_module(
       "__torch__.torch.nn.modules.batchnorm.BatchNorm3d");
 }
 
+bool is_fp16_fp32_cast_op(
+    const Match& match,
+    const std::unordered_map<std::string, Value*>& vmap) {
+  const auto& match_vmap = match.values_map;
+  auto fp16_type = toIValue(match_vmap.at(vmap.at("dtype_fp16")));
+  auto fp32_type = toIValue(match_vmap.at(vmap.at("dtype_fp32")));
+  auto default_param = toIValue(match_vmap.at(vmap.at("default_param")));
```

Comment:
I think it's better to split this check to a separate one, might be useful for other checks as well

Reply:
also, you can name default_param to `%false` so that this check can be used by all values that needs to be false

Reply:
I took a look at https://pytorch.org/docs/stable/tensors.html#torch.Tensor.half, I think it might make more sense to have `half` and `float` here

Reply:
non_blocking sounds irrelevant here.